app/src/main/java/com/github/polypoly/app/ui/game/HeadsUpDisplay.kt

```diff
@@ -315,6 +327,22 @@ fun HudText(name: String, text: String) {
     )
 }
 
+@Composable
+fun MoneyHudText(name: String, text: String) {
+    Box(modifier = Modifier.offset(x = (-50).dp, y = -(7).dp)) {
+        Text(
+            text = text,
+            modifier = Modifier
+                .testTag(name)
+                .background(MaterialTheme.colors.background, shape = Shapes.medium)
+                .width(100.dp)
+                .padding(Padding.medium),
+            fontSize = MaterialTheme.typography.h6.fontSize,
+            fontWeight = MaterialTheme.typography.h6.fontWeight
```

Comment:
you can replace the two lines above by `style = MaterialTheme.typography.h6`

Reply:
fixed